content/blog/shepherding-3-1.md

```diff
@@ -0,0 +1,186 @@
+date = 2019-09-12
+draft = false
+in_search_index = true
+template = "blog_post.html"
++++
+
+For open source projects, it can be difficult to find balance between a number of factors: the free time of contributors, the desires of the communities using the project, the ability to on-ramp and mentor new contributors, and to provide necessary guidance and feedback.
+
+The Rust Projects, and multiple sub-teams and working groups have found success in use of the [RFC Process] for making informed, high level decisions. However, the RFC process does not address the work necessary for exploratory projects before a decision is made, the work necessary to implement made decisions, or the work necessary to maintain and overhaul parts of the projects in the long run.
+
+[RFC Process]: #
+
+To address these needs, I'd like to propose a general concept and approach called **Shepherding**, which aims to assist maintainers of these projects with this aspect of the managing the project. By enumerating core goals and putting a name to concepts, but specifically NOT mandating every aspect of implementation, I hope that this technique can be applied across many teams inside and outside of the Rust Project.
+
+The "Shepherding 3.1" name comes from a blog post written by Niko Matsakis titled [Shepherds 3.0], which also does an excellent job describing the motivations regarding these efforts. This post instead makes a few clarifications and additions to the 3.0 post, and aims to be a "reference manual" for the concept, defining terms and important goals, rather than discussing motivations.
+
+[Shepherds 3.0]: http://smallcultfollowing.com/babysteps/blog/2019/09/11/aic-shepherds-3-0/
+
+<!-- more -->
+
+## Shepherding In Short
+
+Described shortly, Shepherding describes a process of capturing work as tasks, and finding one or more persons responsible for this Task, known as a Shepherd.
+
+By selecting the Shepherding approach, maintainers can provide a consistent, explicitly written process for contributing non-trivial changes to a project, giving a path for potential new contributors.
+
+Shepherding also aims to act as an on-ramp for contributors, giving them a list of projects that need assistance, as well as the ability to suggest improvements, changes, or additions; and to find a shepherd who can assist them with needed mentorship.
+
+Shepherds are expected to be responsible for tracking and guiding the progress of their task, but are not expected to be the sole contributor to this task. Shepherds are expected to typically only be tied to one task at a time, in order to keep all open tasks progressing, as well as to limit the number of tasks open at any given time.
+
+Contributors are expected to take feedback from their shepherd, and to assist in delivering the work described by a single task.
+
+After contributing to a task, the hope is to find new potential shepherds, growing the set of people who feel empowered to assist others in tasks.
```

Comment:
This is kind of what I was getting at, I guess, though I think it could be made more explicit.